package.json

```diff
@@ -25,18 +25,19 @@
     "requirejs": "Reinmar/requirejs"
   },
   "devDependencies": {
-    "@ckeditor/ckeditor5-dev-bundler-rollup": "ckeditor/ckeditor5-dev-bundler-rollup",
-    "@ckeditor/ckeditor5-dev-compiler": "ckeditor/ckeditor5-dev-compiler",
-    "@ckeditor/ckeditor5-dev-env": "^1.0.0",
-    "@ckeditor/ckeditor5-dev-lint": "^1.0.1",
-    "@ckeditor/ckeditor5-dev-utils": "ckeditor/ckeditor5-dev-utils",
     "babel-polyfill": "^6.13.0",
     "benderjs": "^0.4.1",
     "benderjs-chai": "^0.2.0",
     "benderjs-coverage": "benderjs/benderjs-coverage#t/4",
     "benderjs-mocha": "^0.3.0",
     "benderjs-promise": "^0.1.0",
     "benderjs-sinon": "^0.3.0",
+    "@ckeditor/ckeditor5-dev-bundler-rollup": "ckeditor/ckeditor5-dev-bundler-rollup",
```

Comment:
Why did you move all deps here? As far as I noticed, npm automatically sorts them so those starting with `@` are at the beginning (what makes sense).